entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+# shellcheck disable=SC1091
```

Comment:
Shouldnt be linter-disabled-overwrites be all after the shebang in unity? My symetry tick alarm is going off on this ⚠️